src/khoj/main.py

```diff
@@ -15,8 +15,8 @@
 # External Packages
 import uvicorn
 from fastapi import FastAPI
-from PyQt6 import QtWidgets
-from PyQt6.QtCore import QThread, QTimer
+from PySide6 import QtWidgets
+from PySide6.QtCore import QThread, QTimer
```

Comment:
Is this really all you have to do? 😮

Reply:
Surprisingly, yes. It seems like an actual drop in replacement for our use-case